docs/ops/roundtable/index.rst

```diff
@@ -36,10 +36,29 @@ Since ``roundtable`` is a parent app, its ``Application`` resource was not creat
 
 Before bootstrapping the ``roundtable`` app, the ``security`` app needs to be bootstrapped so that Vault is running.
 See :doc:`its documentation <../security/index>` for more information.
-Then, a Vault access token for the Vault Secrets Operator must be created in Vault and stored as the ``vault-secrets-operator`` Kubernetes secret with two keys:
+Then, a Vault access token for the Vault Secrets Operator must be created in Vault and stored as the ``vault-secrets-operator`` Kubernetes secret.
+Here is the template for that secret:
 
-- VAULT_TOKEN
-- VAULT_LEASE_DURATION
+.. code-block:: yaml
+
+   apiVersion: v1
+   kind: Secret
+   metadata:
+     name: vault-secrets-operator
+     namespace: vault-secrets-operator
+   type: Opaque
+   data:
```

Comment:
Instead of `data` in the template, you can use `stringData` ([docs](https://kubernetes.io/docs/concepts/configuration/secret/#creating-a-secret-manually)) instead (this is only available when initially creating the secret). The upside of this is that users don't need to manually encode values: kubectl will base64 encode the values and turn it into a `data` key for you.